### PR TITLE
Adds video

### PR DESCRIPTION
**Pull request by amdcavallaro**
## Description

Adds the NCCO video with an image with a play button added to it. Clicking to it takes you to the video.

### CODE REVIEW

File: _documentation/en/voice/voice-api/guides/ncco.md [hunk 1]
```diff
@@ -88,6 +88,10 @@ source: '_examples/voice/ncco/creating-a-custom-call-or-conversation-for-each-us
 
 ⚓ ringback-tone
 
+Below you an find an One De Minute video explaining what NCCOs are and how to use them:
```
Comment (amdcavallaro) [inline _documentation/en/voice/voice-api/guides/ncco.md:91]:
```suggestion
Below you can find the One Dev Minute video explaining what NCCOs are and how to use them:
```

File: _documentation/en/voice/voice-api/guides/ncco.md [hunk 2]
```diff
@@ -88,6 +88,10 @@ source: '_examples/voice/ncco/creating-a-custom-call-or-conversation-for-each-us
 
 ⚓ ringback-tone
 
+Below you can find the One Dev Minute video explaining what NCCOs are and how to use them:
+[![Here's an One De Minute video explaining what NCCOs are and how to use them](https://user-images.githubusercontent.com/9335603/148561582-4121623a-99c0-445e-b8cf-1f6e1d57d422.png)](https://www.youtube.com/watch?v=26rm0MP2up0)
```
Comment (amdcavallaro) [inline _documentation/en/voice/voice-api/guides/ncco.md:92]:
```suggestion
[![Here's the One Dev Minute video explaining what NCCOs are and how to use them](https://user-images.githubusercontent.com/9335603/148561582-4121623a-99c0-445e-b8cf-1f6e1d57d422.png)](https://www.youtube.com/watch?v=26rm0MP2up0)
```

**Comment by vshisterov** [comment]
Let's move the video to the beginning of the article

**Comment by amdcavallaro** [comment]
Sure @vshisterov. Is there a specific place you prefer?

**Review by abdulajet — APPROVED** [review]

**Pull request merged**